model/Image.php

```diff
@@ -233,10 +247,10 @@ public function SetRatioSize($width, $height) {
 		$heightRatio = $height / $this->height;
 		if( $widthRatio < $heightRatio ) {
 			// Target is higher aspect ratio than image, so check width
-			if($this->isWidth($width) && !Config::inst()->get('Image', 'force_resample')) return $this;
+			if($this->isWidth($width)) return $this;
```

Comment:
Ha, nice update! :) think I was typing a response to your first comment while you wrote this one. Everything you said here makes perfect sense - but I still think that it doesn't apply to this specific case because we're wondering whether it matters if the resampled image filenames change in a few cases, when already *all* the resampled image filenames are being changed in 3.2 before this PR came along. Now I must go to bed :sleeping: